AddressService/AddressService.Repo/Repository.cs

```diff
@@ -266,5 +271,43 @@ public async Task<IEnumerable<string>> GetPostcodesInBoundaryAsync(double swLati
                 return result;
             }
         }
+
+        public async Task<LocationDetails> GetLocationDetails(Location location)
+        {
+            int locationId = (int)location;
+            var locationdetails = _context.Location.Where(x => x.Id == locationId).FirstOrDefault();
+
+            if(locationdetails ==null)
+            {
+                throw new Exception($"Unable to retrieve location for location {location}");
+            }
+
+            LocationInstructions locationInstructions = null;
```

Comment:
I think we might want to simplify the LocationInstructions class in due course, but this will be fine for now.